test/jdk/java/io/IO/IO.java

```diff
@@ -162,7 +163,7 @@ public static Stream<Arguments> args() {
     public void printTest(String mode) throws Exception {
         var file = Path.of(System.getProperty("test.src", "."), "Output.java")
                 .toAbsolutePath().toString();
-        var pb = ProcessTools.createTestJavaProcessBuilder("--enable-preview", file, mode);
+        var pb = ProcessTools.createTestJavaProcessBuilder("-Djdk.console=jdk.internal.le", "--enable-preview", file, mode);
```

Comment:
Have you looked into changing the test so that it runs twice, the second with the jline provider?

Reply:
The child java process `Output.java` calls `System.console()`, so the launcher has to be run with `-Djdk.console=jdk.internal.le`, otherwise NPE is thrown in `Output.java`.

Reply:
Ah okay, no console in the child process for this implementation.